docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -24,15 +24,17 @@ Example for *y-pz* phase space for the *electron* species (``.cfg`` file macro):
 
 The distinct options are (assuming a species ``e`` for electrons):
 
-====================================== ========================================= ============================
+====================================== ======================================================== ============================
 Option                                 Usage                                     Unit
-====================================== ========================================= ============================
-``--e_phaseSpace.period <N>``          calculate each N steps                    *none*
-``--e_phaseSpace.space <x/y/z>``       spatial coordinate of the 2D phase space  *none*
-``--e_phaseSpace.momentum <px/py/pz>`` momentum coordinate of the 2D phase space *none*
-``--e_phaseSpace.min <ValL>``          minimum of the momentum range             :math:`m_\mathrm{species} c`
-``--e_phaseSpace.max <ValR>``          maximum of the momentum range             :math:`m_\mathrm{species} c`
-====================================== ========================================= ============================
+====================================== ======================================================== ============================
+``--e_phaseSpace.period <N>``          calculate each N steps                                   *none*
+``--e_energy.filter``                  Use filtered particles. Available filters are set up in  *none*
```

Comment:
typo:
- `e_energy`

Reply:
Note to my self: do not copy past code snippets